### PR TITLE
certificate_manage wildcard reload

### PR DESCRIPTION
**Pull request by pasimko**
Replaced wildcard cert. Tried writing tests in this one.

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -60,11 +60,11 @@
...
-
-  if node['certificate'].any?
-    include_recipe 'certificate::manage_by_attributes'
-  else
-    include_recipe 'certificate::wildcard'
+  certificate_manage 'wildcard' do
+    cert_file 'wildcard.pem'
+    key_file 'wildcard.key'
+    chain_file 'wildcard-bundle.crt'
+    notifies :reload, 'service[apache2]'
```
Comment (ramereth) [inline recipes/default.rb]: This is the correct service.
```suggestion
    notifies :reload, 'service[dovecot]'
```

File: kitchen.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ provisioner:
   # the ohai_plugin from the dovecot cookbook fails idempotency checks
   # enforce_idempotency: true
   # multiple_converge: 2
-  deprecations_as_errors: true
+  deprecations_as_errors: false
```
Comment (ramereth) [inline kitchen.yml]: You should add a comment here as to which cookbook is causing us to change this.

**Comment by ramereth** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v2.0.2.  Have a nice day!

**Pull request merged**